schutzbot/build.sh

```diff
@@ -19,8 +19,17 @@ if [[ $ID == rhel ]] && ! rpm -q epel-release; then
 fi
 
 # Register RHEL if we are provided with a registration script.
-if [[ -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
-    greenprint "🪙 Registering RHEL instance"
+# if [[ -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
+#     greenprint "🪙 Registering RHEL instance"
+#     sudo chmod +x "$RHN_REGISTRATION_SCRIPT"
+#     sudo "$RHN_REGISTRATION_SCRIPT"
+# fi
+
+# Currently openstack/rhel-8.4-x86_64 cannot subcribe, disable subscription.
+# TODO: remove this after openstack/rhel-8.4-x86_64 can subscribe
+echo "Do not subscribe for rhel84."
+if [[ $ID == "rhel" && $VERSION_ID == "8.3" && -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
```

Comment:
I am wondering if this condition can be based not on the specific OS version but the fact that it is a BETA release ?

Reply:
Agree, it should be based on BETA release.
I have no idea how to check if RHEL is BETA, I have created a jira task for it.
https://issues.redhat.com/browse/COMPOSER-1034

Reply:
The information is already contained inside the /etc/os-release file. We don't need a separate ticket for that.

Reply:
Fixed, thanks.